installing/installing_openstack/installing-openstack-nfv-preparing.adoc

```diff
@@ -0,0 +1,45 @@
+:_content-type: ASSEMBLY
+[id="installing-openstack-nfv-preparing"]
+= Preparing to install a cluster that uses SR-IOV or OVS-DPDK on OpenStack
+include::_attributes/common-attributes.adoc[]
+:context: installing-openstack-nfv-preparing
+
+toc::[]
+
+Before you install an OCP cluster that uses single-root I/O virtualization (SR-IOV) or Open vSwitch with the Data Plane Development Kit (OVS-DPDK) on {rh-openstack-first}, you must understand the requirements for each technology and then perform preparatory tasks.
+
+include::modules/installation-openstack-nfv-requirements.adoc[leveloffset=+1]
+include::modules/installation-openstack-sr-iov-requirements.adoc[leveloffset=+2]
+include::modules/installation-openstack-ovs-dpdk-requirements.adoc[leveloffset=+2]
+
+[id="installing-openstack-nfv-preparing-tasks-sr-iov"]
+== Preparing to install a cluster that uses SR-IOV
+
+You must configure {rh-openstack} before you install a cluster that uses SR-IOV on it. 
+
+include::modules/installation-osp-configuring-sr-iov.adoc[leveloffset=+2]
+
+[id="installing-openstack-nfv-preparing-tasks-ovs-dpdk"]
+== Preparing to install a cluster that uses OVS-DPDK
+
+You must configure {rh-openstack} before you install a cluster that uses SR-IOV on it. 
+
+* Complete link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.2/html/network_functions_virtualization_planning_and_configuration_guide/part-dpdk-configure#p-ovs-dpdk-flavor-deploy-instance[Creating a flavor and deploying an instance for OVS-DPDK] before you install a cluster on RHOSP.
+
+After you perform pre-installation tasks, install your cluster by following the most relevant {product-title} on {rh-openstack} installation instructions. Then, perform the tasks under "Next steps" on this page.
```

Comment:
This makes for an awkward user flow because the assembly does not track a single user goal. (These aren't steps to prepare to install your cluster and, ideally, would go somewhere else. The first three sections could be an assembly, and the last section could be additional resources for that assembly.)

Reply:
I think of the assembly goal as "plan gritty OSP things and then prepare to deploy a cluster that uses NFV stuff," broadly. That prep is distinct from other installation prep tasks in our docs. 

I do think of this as an ongoing reorganization, if that makes this seem less groady. It's a break from lengthy installation assemblies that could never really be complete, anyway--lots of "ifs" and OSP handwaving--though it's not entirely clean, yet.

Reply:
I think it'll be fine as a step toward a saner organization

Reply:
It should actually happen, too, as how to make this all less hairy is a topic in ShiftStack eng.